products/cpdb/cpdb/projects/metadata.yml

```diff
@@ -0,0 +1,397 @@
+name: projects
+display_name: Capital Projects Database (CPDB) - Projects
+summary: ""
+description: >
+  The Capital Projects Database reports information at the project level
+  on discrete capital investments from the Capital Commitment Plan.Each
+  row is uniquely identified by its Financial Management Service (FMS) ID,
+  and contains data pertaining to the sponsoring and managing agency.
+
+  
+  To explore the data, please visit 
+  <a href="https://capitalplanning.nyc.gov/map#10/40.7128/-74.0807">Capital Planning Explorer</a>
+
+
+  For additional information, please visit 
+  <a href="https://www.ibo.nyc.ny.us/iboreports/IBOCBG.pdf">A Guide to The Capital Budget</a>
+  
+tags: [capital projects database, capital planning, cpdb, budget, omb]
+each_row_is_a: Capital Project
+
+destinations:
+  - id: socrata_projects
+    type: socrata
+    four_four: "fi59-268w"
+    datasets:
+      - projects
+
+  - id: socrata_projects_points
+    type: socrata
+    four_four: "h2ic-zdws"
+    datasets:
+      - projects_points
+
+  - id: socrata_projects_polygons
+    type: socrata
+    four_four: "9jkp-n57r"
+    datasets:
+      - projects_polygons
+
+package:
+  dataset_files:
+    - name: projects
+      type: csv
+      filename: cpdb_projects.csv
+      overrides:
+        omit_columns: [geometry]
+
+    - name: projects_points
+      type: shapefile
+      filename: cpdb_projects_pts.shp.zip
+      overrides:
+        display_name: Capital Projects Database (CPDB) - Projects (Points)
+        omit_columns: []
+        ignore_validation: []
+        columns:
+          maprojid: {name: maprojid}
```

Comment:
Should we leave out the ones that don't override anything? Or do we think it's better to include all columns like this for consistency. Could go either way

Reply:
I don't know why I thought we had to put these (@alexrichey?). I'd rather leave em out

Reply:
I just tried to publish without em and got 

```bash
│ /Users/damonmccullough/github/data-engineering/dcpy/connectors/socrata/publish.py:50 in          │
│ _socrata_request                                                                                 │
│                                                                                                  │
│    47 │   request_fn = getattr(requests, method.lower())                                         │
│    48 │   resp = request_fn(url, auth=_simple_auth(), **kwargs)                                  │
│    49 │   if not resp.ok:                                                                        │
│ ❱  50 │   │   raise Exception(resp.text)   

...

Exception: {"params":{},"message":"You are not allowed to access that resource","key":"forbidden"}
```